ticdc: update time-zone, add without valid index replic, update cdc c… (#4163)
pingcap/docs: ticdc/ticdc-overview.md

    - Every column of the index is explicitly defined as non-nullable (`NOT NULL`).
    - The index does not have the virtual generated column (`VIRTUAL GENERATED COLUMNS`).

> Since v4.0.8, TiCDC supports replicating the table **without a valid index** by modifying the task configuration. However, the guarantee of data consistency will be compromised to some extent. For more details, see [Replicate the table without a valid index](/ticdc/manage-ticdc.md#replicate-the-table-without-a-valid-index).

Reviewer: ```suggestion
Since v4.0.8, TiCDC supports replicating tables **without a valid index** by modifying the task configuration. However, this compromises the guarantee of data consistency to some extent. For more details, see [Replicate tables without a valid index](/ticdc/manage-ticdc.md#replicate-tables-without-a-valid-index).
```
